stable/datacube-dev-env/values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for datacube-wms.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+environment: dev
+image:
+  registry: docker.io
+  repository: opendatacube/k8s-dev
+  tag: "latest"
+  pullPolicy: Always
+
+datacube:
+  configPath: /opt/odc/datacube.conf
+
+database:
+  create: true
```

Comment:
Better to keep the defaults  set to False

Reply:
Updated